salt/edx/prod.sls

```diff
@@ -34,11 +34,10 @@ configure_python_ppa_for_edx:
 install_os_packages:
   pkg.installed:
     - pkgs:
-        - git
+        - git: 2.9.2-0ppa1~ubuntu12.04.1
         - libmysqlclient-dev
-        - python-dev
```

Comment:
I had added that when battling the scipy issue. Ansible is supposed to be installing 2.7.10

On Aug 9, 2016 5:43 PM, Brandon DeRosier <notifications@github.com> wrote:

In salt/edx/prod.sls<https://github.com/mitodl/salt-ops/pull/150#discussion_r74150293>:

>          - libmysqlclient-dev
> -        - python-dev


Shouldn't this be set to 2.7.3 - or is that not the version installed by Ansible?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/mitodl/salt-ops/pull/150/files/e31548d87ed101d0dc058ca036061c268a386bd8#r74150293>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AAdPcPfj68oagHEfw2kq5LvyaO3lrE26ks5qePRSgaJpZM4JgeSL>.